src/Microsoft.ML.TimeSeries/Forecast.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// Interface for forecasting models.
+    /// </summary>
+    /// <typeparam name="T">The type of values that are forecasted.</typeparam>
+    public interface ICanForecast<out T>
+    {
+        /// <summary>
+        /// Train a forecasting model from an <see cref="IDataView"/>.
+        /// </summary>
+        /// <param name="dataView">Reference to the <see cref="IDataView"/></param>
+        /// <param name="inputColumnName">Name of the input column to train the forecasing model.</param>
+        void Train(IDataView dataView, string inputColumnName);
```

Comment:
>inputColumnName [](start = 46, length = 15)

I think these methods should be more similar to the `IEstimator` methods, in the sense that they should not take the input column names at training time, but at construction time.